src/types.ts

```diff
@@ -221,3 +221,22 @@ export type Voucher = {
   serial: string;
   denomination: number;
 };
+
+export const Statistics = t.intersection([
```

Comment:
Does this need to be exported?

Reply:
yup! im currently using this in `getStatistics` !

Reply:
Hmm but that service relies on `DailyStatistics` rather than `Statistics`

Reply:
ahhh yes! got it, removing `Statistics`!